lib/media/ttml_text_parser.js

```diff
@@ -331,6 +326,20 @@ shaka.media.TtmlTextParser.addStyle_ = function(
       }
     }
   }
+
+  var align = TtmlTextParser.getStyleAttribute_(
+      cueElement, region, styles, 'tts:textAlign');
+  if (align) {
+    if (align == 'center') {
+      // Need to set cue.position to auto to get centered
+      // subtitles
+      cue.align = 'middle';
+      cue.position = 'auto';
```

Comment:
Sorry, forgot to answer the sample request. I don't think it will be possible to share due to drm and geoblocking issues. However if you for example set cue.align=middle and cue.position=2 I believe you would get the same effect as some TTML would produce. Again, not necessarily that part is a bug in Chrome and maybe how it is intended.